utils/bot.ts

```diff
@@ -0,0 +1,83 @@
+import { timeRegexPattern } from './const'
+export const registerRide = () => {}
+
+export const getPureCommand = (command: string) => {
+  if (command.indexOf('@carona_v2_bot') > -1) {
+    return command.split('@')[0].toLowerCase()
+  }
+  return command.toLowerCase()
+}
+
+export const validateTimeFormat = (time: string) => {
+  let matches = timeRegexPattern.exec(time)
+  if (!matches) return [false, null] as const
+
+  return [true, matches] as const
+}
+
+export const getRideInfo = (fields: string[]) => {
+  let isToday = false
+  let time
+  let description
+
+  if (fields[1].trim() === 'hoje') {
+    isToday = true
+    time = fields[2]
+    description = fields.slice(3, fields.length).join(' ')
+  } else {
```

Comment:
I don`t remember why we added this logic, I think it was justfor debugging purposes